src/algebra/group/pi.lean

```diff
@@ -147,12 +147,22 @@ end mul_hom
 
 section mul_hom
 
-/-- A family of monoid homomorphisms `f a : γ →ₙ* β a` defines a monoid homomorphism
-`pi.monoid_mul_hom f : γ →ₙ* Π a, β a` given by `pi.monoid_mul_hom f x b = f b x`. -/
-@[to_additive "A family of additive monoid homomorphisms `f a : γ →ₙ+ β a` defines a monoid
-homomorphism `pi.monoid_add_hom f : γ →ₙ+ Π a, β a` given by `pi.monoid_add_hom f x b
+/-- A family of monoid homomorphisms `f a : γ →* β a` defines a monoid homomorphism
+`pi.monoid_mul_hom f : γ →* Π a, β a` given by `pi.monoid_mul_hom f x b = f b x`. -/
+@[to_additive "A family of additive monoid homomorphisms `f a : γ →+ β a` defines a monoid
+homomorphism `pi.monoid_add_hom f : γ →+ Π a, β a` given by `pi.monoid_add_hom f x b
 = f b x`.", simps]
-def pi.monoid_mul_hom {γ : Type w} [Π i, has_mul (f i)] [has_mul γ]
+def pi.monoid_mul_hom {γ : Type w} [Π i, monoid (f i)] [monoid γ]
```

Comment:
This should be `pi.monoid_hom` :)

Reply:
I see, a monoid is multiplicative by default..